### PR TITLE
Change dataset env var config

### PR DESCRIPTION
**Pull request by rmn-boiko**
Change dataset env var config

### CODE REVIEW

File: src/adapter/graphql/src/utils.rs [hunk 1]
```diff
@@ -114,10 +114,10 @@ pub(crate) async fn get_task(
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 pub(crate) fn ensure_dataset_env_vars_mode(ctx: &Context<'_>) -> Result<(), GqlError> {
```
Comment (s373r) [inline src/adapter/graphql/src/utils.rs:116]: what about `ensure_dataset_env_vars_enabled()`?

Comment (rmn-boiko) [reply src/adapter/graphql/src/utils.rs:116]: Yeah, good catch. Thanks

File: src/domain/datasets/domain/src/entities/dataset_env_var.rs [hunk 2]
```diff
@@ -232,43 +232,46 @@ impl std::error::Error for AesGcmError {}
...
+        }
+        false
+    }
 }
 
 impl Default for DatasetEnvVarsConfig {
     fn default() -> Self {
         Self {
-            mode: Some(DatasetEnvVarsType::Static),
+            enabled: Some(false),
```
Comment (sergiimk) [inline src/domain/datasets/domain/src/entities/dataset_env_var.rs:274]: If we default to `Some(false)` then the feature is disabled by default and we will never warn a user that it was not configured?

Comment (rmn-boiko) [reply src/domain/datasets/domain/src/entities/dataset_env_var.rs:274]: yeah, changed

**Review by zaychenko-sergei — APPROVED** [review]

**Pull request merged**